projects/packages/forms/src/contact-form/css/grunion.css

```diff
@@ -18,6 +18,36 @@
 	transition: opacity 0.3s ease-out;
 }
 
+.jetpack-form-step {
+	position: absolute;
+	z-index: -1;
+	opacity: 0;
+	transition: transform 1s cubic-bezier(0.53, 0, 0, 1), opacity 0.5s cubic-bezier(0.53, 0, 0, 1);
```

Comment:
Could be wrapped in `@media not (prefers-reduced-motion) { }` to avoid showing animations to those with "prefers reduced motion" setting in their browser.